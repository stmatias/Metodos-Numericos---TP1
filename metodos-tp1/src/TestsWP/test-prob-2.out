0.333333
0.333333
1
0.4
0.4
0.666667
