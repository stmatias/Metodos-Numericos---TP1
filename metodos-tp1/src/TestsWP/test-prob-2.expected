0.33333
0.33333 
1.00
0.40  
0.40  
0.66667 
